package.json

```diff
@@ -34,11 +34,11 @@
   "dependencies": {
     "async": "^1.5.2",
     "base64url": "^3.0.0",
-    "bunyan": "^1.8.0",
+    "bunyan": "^1.8.14",
     "cache-manager": "2.10.2",
     "https-proxy-agent": "^2.2.2",
-    "jws": "^3.1.3",
     "jwk-to-pem": "^1.2.6",
```

Comment:
Can you also update the version of this package as this version in use depends on an insecure version of elliptic, thanks.

![image](https://user-images.githubusercontent.com/441581/91956069-07808b80-ecfc-11ea-8b7e-a99d014cbd28.png)

Reply:
Done, published as `passport-azure-ad@4.3.0-beta.1`, let me know if you have any issues.